File: MyMovie/app/src/main/java/com/test/mymovie/MainActivity.java
package com.test.mymovie;

import androidx.activity.result.ActivityResult;
import androidx.activity.result.ActivityResultCallback;
import androidx.activity.result.ActivityResultLauncher;
import androidx.activity.result.contract.ActivityResultContracts;
import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.content.Intent;
import android.os.Bundle;

import com.test.mymovie.databinding.ActivityMainBinding;
import com.test.mymovie.service.NaverService;
import com.test.mymovie.service.impl.NaverMovieServiceImplV1;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding mainBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mainBinding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(mainBinding.getRoot());

        NaverService naverService = new NaverMovieServiceImplV1(mainBinding.movieListView);

        naverService.getMovies("1987");

    }
}

File: ThreeDays/app/src/main/java/com/callor/threedays/MainActivity.kt
package com.callor.threedays

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.callor.threedays.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding:ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnLogin.setOnClickListener() {
//            binding.loginLayoutId
        }
    }

}